google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/ProjectDebuggeeBinding.java

```diff
@@ -87,13 +91,13 @@ public void treeStructureChanged(TreeModelEvent e) {
 
   @NotNull
   public CloudDebugProcessState buildResult(Project project) {
-    Long number = myElysiumProjectId.getProjectNumber();
+    Long number = elysiumProjectId.getProjectNumber();
     String projectNumberString = number != null ? number.toString() : null;
-    DebugTarget selectedItem = (DebugTarget)myDebugeeTarget.getSelectedItem();
+    DebugTarget selectedItem = (DebugTarget) debugeeTarget.getSelectedItem();
```

Comment:
You mean the name? Yea, I wasn't sure myself either - this came from preexisting code. I'll change it to moduleSelector which is what it really is.